src/components/OptionRow.tsx

```diff
@@ -326,7 +318,7 @@ export default React.memo(
         prevProps.showSelectedState === nextProps.showSelectedState &&
         prevProps.highlightSelected === nextProps.highlightSelected &&
         prevProps.showTitleTooltip === nextProps.showTitleTooltip &&
-        !_.isEqual(prevProps.option.icons, nextProps.option.icons) &&
+        !lodashIsEqual(prevProps.option.icons, nextProps.option.icons) &&
```

Comment:
Please fix conflict.
`!lodashIsEqual` > `lodashIsEqual` to fix https://github.com/Expensify/App/issues/33353

Reply:
@situchan Done, thank you!